src/Facets/GlacisFacet.sol

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+import { ILiFi } from "../Interfaces/ILiFi.sol";
+import { LibAsset, IERC20, SafeERC20 } from "../Libraries/LibAsset.sol";
+import { LibSwap } from "../Libraries/LibSwap.sol";
+import { ReentrancyGuard } from "../Helpers/ReentrancyGuard.sol";
+import { SwapperV2 } from "../Helpers/SwapperV2.sol";
+import { Validatable } from "../Helpers/Validatable.sol";
+import { IGlacisAirlift } from "../Interfaces/IGlacisAirlift.sol";
+
+/// @title Glacis Facet
+/// @author LI.FI (https://li.fi/)
+/// @notice Integration of the Glacis airlift (wrapper for native token bridging standards)
+/// @custom:version 1.0.0
+contract GlacisFacet is ILiFi, ReentrancyGuard, SwapperV2, Validatable {
+    /// Storage ///
+
+    /// @notice The contract address of the glacis airlift on the source chain.
+    IGlacisAirlift public immutable airlift;
+
+    /// Types ///
+
+    /// @param refund Refund address
+    /// @param nativeFee The fee amount in native token required by the Glacis Airlift.
+    struct GlacisData {
+        address refund;
```

Comment:
for consistency to other facets I would suggest to rename this variable to "refundAddress" and better parameter description